apps/client/src/app/services/import-activities.service.ts

```diff
@@ -197,6 +201,22 @@ export class ImportActivitiesService {
     };
   }
 
+  private parseComment({
```

Comment:
Can you please move this fuction up ⬆️ to keep the alphabetical order? (before `parseCurrency()`)